src/components/Textarea/Textarea.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { axe, render, screen } from 'testUtils';
+import Textarea from './Textarea';
+
+test('renders the Textarea correctly', () => {
+  render(<Textarea></Textarea>);
+  expect(screen.getByRole('textbox')).toBeInTheDocument();
+});
+test('renders the Textarea correctly with place holder', () => {
+  render(
+    <Textarea placeholder="Hello World" data-testid="dataTestId"></Textarea>,
+  );
+  expect(screen.getByRole('textbox')).toBeInTheDocument();
+  expect(screen.getByTestId('dataTestId')).toBeInTheDocument();
+});
```

Comment:
```suggestion
});

test('renders the Textarea correctly with place holder', () => {
  render(
    <Textarea placeholder="Hello World"></Textarea>,
  );
  expect(screen.getByRole('textbox')).toBeInTheDocument();
  expect(screen.getByPlaceholderText('Hello World')).toBeInTheDocument();
});
```
You can replace `data-testid` with `getByPlaceholderText`

Reply:
done